### PR TITLE
first draft for contribution guide

### PR DESCRIPTION
**Pull request by kiryteo**
## Discussion required

Added the guide for contributing to lumin. This is only a primer for a better and detailed version and can be improved upon further discussion.

### CODE REVIEW

**Comment by GilesStrong** [comment]
Hi @kiryteo sorry for the wait, and thanks again for writing this. There's just one change, which I'll note in the review.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+```bash
+conda uninstall lumin
+pip uninstall lumin
+```
+
+2. Clone a copy of PyTorch from source:
+
+```bash
+git clone https://github.com/GilesStrong/lumin
+cd pytorch
```
Comment (GilesStrong) [inline CONTRIBUTING.md:31]: Could you correct this to `cd lumin`, thanks

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+```
+
+3. Perform the modifications in source code and rebuild, test the changes locally.
+
+```bash
+python setup.py install (after the changes)
```
Comment (GilesStrong) [inline CONTRIBUTING.md:37]: Could you move "(after the changes)" outside of the bash line

File: CONTRIBUTING.md (hunk 1 above)
Comment (kiryteo) [reply CONTRIBUTING.md:31]: Sorry, referred to PyTorch guide and thus this mishap, will correct.

File: CONTRIBUTING.md (hunk 2 above)
Comment (kiryteo) [reply CONTRIBUTING.md:37]: Sure.

**Comment by kiryteo** [comment]
@GilesStrong  Modification done, you can take a look

**Comment by GilesStrong** [comment]
Perfect! Merging. Thanks again!

**Pull request merged**